build

```diff
@@ -10,10 +10,19 @@ try {
 var child_process = require('child_process');
 var fs = require('fs');
 var path = require('path');
+var replace = require('replace');
 
 function shell(cmd) { child_process.execSync(cmd, {stdio: 'inherit', cwd: __dirname}); }
 function sucrase(src, out) {
-  shell(`npx sucrase -q ${src} -d ${out} --transforms typescript,imports --enable-legacy-typescript-module-interop`);
+	shell(`npx sucrase -q ${src} -d ${out} --transforms typescript,imports --enable-legacy-typescript-module-interop`);
+}
+function rewrite(src, out, dist) {
+	replace({
+		regex: `(require\\\(.*?)(${src})(.*?\\\))`,
+		replacement: `$1${out}$3`,
+		paths: fs.readdirSync(dist).map(f => path.join(__dirname, dist, f)),
```

Comment:
Is the `readdirSync` necessary? `replace` supports folders...

Reply:
> `replace` supports folders...

Does it though? Passing a directory name doesn't work for me (even when using `recursive`, which also makes it dirt slow).

Reply:
Well, its documentation says it does. You should maybe report a bug if it doesn't.

Reply:
I was holding it wrong by passing in the directory as a `string` instead of a `[string]`, and Javascript being nice and untyped was happily recursively from `/` etc. https://github.com/Zarel/Pokemon-Showdown/pull/5245